include/proxy-wasm/wasm.h

```diff
@@ -91,6 +94,12 @@ class WasmBase : public std::enable_shared_from_this<WasmBase> {
     return nullptr;
   }
 
+  // Return true if the named ABI function is allowed to be exposed to the module.
+  bool abiFunctionAllowed(std::string abi_function_name) {
```

Comment:
This should probably be called `capabilityAllowed` or something, to avoid tying this to the ABI function names.

Reply:
Fixed.